### ISSUE TITLE
Press the R key twice the development server returned response error code: 404

### ISSUE THREAD

**Issue by chelseaisgood** [chelseaisgood/react-native-albums#2]

**Comment by chelseaisgood** [comment]
Only happens when doing run-android. Running run-ios does not meet with this problem.

**Comment by chelseaisgood** [comment]
Resolved. The reason is that another application takes port 8081, which is needed for running Metro Bundler. After doing some research, I found out someone does "react-native start" to start the bundler before running "react-native run-ios/android". Running "react-native run-ios" is successful when I previously started the Bundler. After running "react-native run-android", the app can be viewed. However, after could click R to reload the app, it shows 404. In the window which started bundler, it does not show something like this:

BUNDLE  [android, dev] ./index.js ▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓ 100.0% (481/481), done.

So this is why the app cannot be reloaded.

So if nothing is holding the port 8081, every time when we type "react-native run-ios/android", the Bundler can be automatically started.

When the app is loaded, the window of Bundler should have a progress bar showing the detail of loading the bundle file under ios/android.
<img width="573" alt="screen shot 2018-03-03 at 9 52 46 pm" src="https://user-images.githubusercontent.com/16313161/36941606-47f21dae-1f2d-11e8-85d7-0463f343f4e1.png">
